locales/en/landing.ftl

```diff
@@ -0,0 +1,44 @@
+# Landing page
+
+## Hero
+
+find-out-if-breached = Find out if you’ve been in a data breach
+stay-safe-with-tool = Stay safe with a simple yet powerful tool that tracks whether your personal information has been exposed in a data breach. We’ll help you understand what to do next and continuously monitor for any new breaches — all at no cost to you.
+sign-up-free = Sign up for free
+
+## Why use Firefox Monitor?
+
+why-use-monitor = Why use Firefox Monitor?
+identifying-breaches = Identifying and resolving data breaches is an important step in ensuring your online privacy.
+protect-account = Protect your accounts
+protect-account-prevent-hackers = Prevent hackers from getting into your accounts with breached passwords.
+prevent-fraud = Prevent fraud & cybercrimes
```

Comment:
I think the content guidelines suggest to stay away from &
https://mozilla-hub.atlassian.net/wiki/spaces/FIREFOX/pages/11043269/Punctuation#Punctuation-Ampersand(%26)